pkgs/applications/misc/yate/default.nix

```diff
@@ -28,11 +28,13 @@ composableDerivation.composableDerivation {} ( fixed : {
         -e 's@-Wl,--retain-symbols-file@@'
     '';
 
-  meta = { 
+  meta = {
     description = "YATE - Yet Another Telephony Engine";
```

Comment:
Could you also remove the package name from `description`, so that it adheres to http://nixos.org/nixpkgs/manual/#idm140737319319584?